docs/development/build.md

```diff
@@ -14,24 +14,18 @@ After setting up Go, you should be able to `go get` cAdvisor as expected (we use
 $ go get -d github.com/google/cadvisor
 ```
 
-We use `godep` so you will need to get that as well:
-
-```
-$ go get github.com/tools/godep
-```
-
 ## Building from Source
 
 At this point you can build cAdvisor from the source folder:
 
 ```
-$GOPATH/src/github.com/google/cadvisor $ godep go build .
+$GOPATH/src/github.com/google/cadvisor $ make
```

Comment:
Depends on the context I think. We definitely want contributors to run `make all` before posting patches. But it is not necessary for building though.